packages/models-library/src/models_library/project_networks.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Dict
+
+from models_library.projects import ProjectID
+from pydantic import BaseModel, Field, constr, validate_arguments
+
+from .generics import DictKey, DictModel, DictValue, Generic
+from .projects_nodes_io import NodeIDStr
+
+SERVICE_NETWORK_RE = r"^[a-zA-Z]([a-zA-Z0-9_-]{0,63})$"
+
+PROJECT_NETWORK_PREFIX = "prj-ntwrk"
+
+DockerNetworkName = constr(regex=SERVICE_NETWORK_RE)
+DockerNetworkAlias = constr(regex=SERVICE_NETWORK_RE)
+
+
+@validate_arguments
+def validate_network_name(value: DockerNetworkName) -> DockerNetworkName:
+    return value
+
+
+@validate_arguments
+def validate_network_alias(value: DockerNetworkAlias) -> DockerNetworkAlias:
+    return value
+
+
+class BaseModelDict(DictModel, Generic[DictKey, DictValue]):
+    def dict(self, *args, **kwargs) -> Dict[str, Any]:
```

Comment:
I will need also explanations for this one.
if you use json() instead of dict() then the __root__ key is also removed.

Reply:
Managed to remove this and use your suggestion.